src/util/rust_util/mod.rs

```diff
@@ -78,6 +78,19 @@ impl<T> InitializeOnce<T> {
         debug_assert!(self.once.is_completed());
         unsafe { (*self.v.get()).assume_init_ref() }
     }
+
+    /// Get a mutable reference to the value.
+    /// This is currently only used for SFTMap during plan creation (single threaded),
+    /// and before the plan creation is done, the binding cannot use MMTK at all.
```

Comment:
Yes.  This is an evidence that VMMap should be owned by an MMTk instance so that we have mutable access to the VMMap instance when creating MMTk and the concrete Plan.  It should be part of the refactoring discussed in https://github.com/mmtk/mmtk-core/issues/932